data/companies.json

```diff
@@ -32,6 +64,54 @@
         "youtube": null
       }
     },
+    {
+      "name": "Amnil Technologies Pvt. Ltd.",
+      "establishedAt": 1230747300,
+      "location": "Jhamsikhel, Lalitpur, Nepal",
+      "website": "https://www.amniltech.com/",
+      "email": "info@amniltech.com",
+      "phone": "+977-15431784",
+      "socials": {
+        "facebook": "https://www.facebook.com/amniltech",
+        "github": null,
+        "instagram": null,
+        "linkedin": "https://www.linkedin.com/company/amnil-technologies-pvt-ltd-/",
+        "twitter": "https://twitter.com/amniltech",
+        "youtube": null
+      }
+    },
+    {
+      "name": "Aspark Systems",
+      "establishedAt": 1609438500,
+      "location": "Mid Baneshwor, Kathmandu, Nepal",
+      "website": "https://asparksys.com/",
+      "email": "info@asparksys.com",
+      "phone": "+977-9820805602",
+      "socials": {
+        "facebook": "https://www.facebook.com/asparksystems",
+        "github": "https://github.com/asparksys",
+        "instagram": "https://www.instagram.com/asparksystems/",
+        "linkedin": "https://www.linkedin.com/company/asparksystems/",
+        "twitter": null,
+        "youtube": null
+      }
+    },
+    {
+      "name": "Awecode",
+      "establishedAt": 1388513700,
+      "location": "Mahadevsthan, Koteshwar, Kathmandu, Nepal",
+      "website": "https://awecode.com/",
+      "email": "hi@awecode.com",
+      "phone": null,
```

Comment:
Also we need the phone number here.

Reply:
No Phone no. info info on their site. However, adding the number according to info provided in their vacancy post: https://www.linkedin.com/posts/awecode_join-our-team-now-hiring-flutter-intern-activity-7084492296359936002-nk80?utm_source=share&utm_medium=member_desktop.

Reply:
Also @thenishantsapkota , for some companies with more than 1 phone numbers, the `phone` key should be `list` type !